mixin/rows.libsonnet

```diff
@@ -0,0 +1,60 @@
+local g = import './g.libsonnet';
+
+{
+  new(this):
+    {
+      local panels = this.grafana.panels,
+
+      overview:
+        g.panel.row.new('Overview')
+        + g.panel.row.withCollapsed(false)
+        + g.panel.row.withPanels(
+          [
+            panels.overviewWarehouseActivity { gridPos: { w: 12 } },
+            panels.accountStorageUsage { gridPos: { w: 12 } },
+            panels.loginAttempts { gridPos: { w: 12 } },
+            panels.successfulLoginAttempts { gridPos: { w: 6 } },
+            panels.failedLoginAttempts { gridPos: { w: 6 } },
+          ]
+        ),
+
+      billing:
+        g.panel.row.new('Billing')
+        + g.panel.row.withPanels(
+          [
+            panels.averageHourlyCreditsUsed { gridPos: { w: 12 } },
+            panels.autoclusteringCreditsUsed { gridPos: { w: 12 } },
+            panels.top5ServiceComputeCreditsUsed { gridPos: { w: 12 } },
+            panels.top5ServiceCloudServiceCreditsUsed { gridPos: { w: 12 } },
+            panels.top5WarehouseComputeCreditsUsed { gridPos: { w: 12 } },
+            panels.top5WarehouseCloudServicesCreditsUsed { gridPos: { w: 12 } },
+            panels.top5DatabaseAutoclusteringCreditsUsed { gridPos: { w: 12 } },
+            panels.top5TableAutoclusteringCreditsUsed { gridPos: { w: 12 } },
+          ]
+        ),
+
+      dataOwnership:
+        g.panel.row.new('Data ownership')
+        + g.panel.row.withPanels(
+          [
+            panels.activeSchemaOwnedData { gridPos: { w: 6 } },
+            panels.failsafeSchemaOwnedData { gridPos: { w: 6 } },
+            panels.cloneSchemaOwnedData { gridPos: { w: 6 } },
```

Comment:
These panels all have pretty sizeable legends in the sample screenshots.
I think they would benefit from right-aligned table legends and being 12 wide like we do when legends start scrolling.

What do you think?